tests/test_visitors/test_ast/test_builtins/test_strings/test_formatted_string.py

```diff
@@ -16,8 +22,33 @@
 some.format(2)
 """
 
+# Allowed
+f_single_chained_attr = "f'{attr1.attr2}'"
+f_variable_lookup = "f'smth {value}'"
+f_dict_lookup_str_key = "f'smth {dict_value[\"key\"]}'"
+f_list_index_lookup = "f'smth {list_value[0]}'"
+f_function_empty_args = "f'smth {user.get_full_name()}'"
+f_attr_on_function = "f'{fcn().attr}'"
+f_single_chained_functions = "f'{f1().f2()}'"
+f_calling_returned_function = "f'{calling_returned_function()()}'"
+f_double_indexing = "f'{list[0][1]}'"
```

Comment:
Should double indexes be allowed? I though we were explicitly allowing only single access operations. 

See discussion at https://github.com/wemake-services/wemake-python-styleguide/pull/1315/files#r405069203

Reply:
We debated this and found it readable. But if you prefer this to be disallowed then it will be fixed.

Reply:
Yes, that's true.

Reply:
Just to clarify, double indexing and double function calls should NOT be allowed?

Reply:
They should not be allowed. Only single level of chaining is fine.